platform-config.yml

```diff
@@ -0,0 +1,69 @@
+# This file is just here to support the tests.
```

Comment:
It can't unfortunately. Because click works using decorators, get_environment_pipeline_names() which is used in the Choice in the cli option "--pipeline" gets called at compile-time not runtime, and so platform-config.yml needs to be in the "project root" in order for the choices to get populated. We can't set this up in tests as it's already been compiled at that point.

If you can think of a good way round this I'm all ears, as I'm not particularaly happy about it either. Also means we can't test that the options are coming through with anything other than the values from that file which isn't ideal either...

For context - I want to demo click cli autocompletion next week hopefully so that you can tab-complete pipelines. If it works well we can roll it out to other commands to start to address some of the complaints people have around the cli tool usability.

Reply:
Could rename the file to `test-platform-config` just to signpost it? If that's not possible, it's definitely not a blocker for me

Reply:
The tests around this for `platform-helper version get-required-platform-helper --pipeline` wouldn't be able to find platform-config.yml at compile time and so the valid pipeline values will always be empty. Can't rename "test-platform-config" on the fly as it's too late.